frontend/src/features/admin-form/create/builder-and-design/utils/getMutationErrorMessage.ts

```diff
@@ -0,0 +1,13 @@
+import { StatusCodes } from 'http-status-codes'
+
+import { HttpError } from '~services/ApiService'
+
+export const getMutationErrorMessage = (error: Error): string => {
+  if (!(error instanceof HttpError)) return error.message
+  switch (error.code) {
+    case StatusCodes.FORBIDDEN:
+      return 'You do not have permission to modify this form.'
+    default:
+      return error.message
```

Comment:
nit: feel like it'd be easier to understand without the negation, like in [admin-form/common/mutation](https://github.com/opengovsg/FormSG/blob/form-v2/develop/frontend/src/features/admin-form/common/mutations.ts#L134-L153)
```suggestion
  if (error instanceof HttpError) {
    switch (error.code) {
      case StatusCodes.FORBIDDEN:
        return 'You do not have permission to modify this form.'
      default:
        return error.message
 } 
 return error.message
```

Reply:
[90904e8](https://github.com/opengovsg/FormSG/pull/4829/commits/90904e822d5db8485d7585a35227795b309648d4)